### PR TITLE
chore(integration):integrate a working fsm

### PR DESCRIPTION
**Pull request by comptonben**

### CODE REVIEW

**Review by DavidTou — COMMENTED** [review]
Looks good

File: uPy/main.py [hunk 1]
```diff
@@ -113,13 +130,15 @@
             if onet[0] not in ap_blacklist:
                 # Try to connect to WiFi access point
                 apSSID = onet[0]
-                print ("Connecting to "+str(onet[0],"utf-8")+" ...\n")
```
Comment (comptonben) [inline uPy/main.py]: add this back in and add a #TODO

File: uPy/post.py [hunk 2]
```diff
@@ -17,14 +17,18 @@
...
             print("Post Request Successful")
+            logger.info("Post Request Successful")
             return True
         else:
+            #TODO: remove print
             print("Post Request Unsuccessful")
+            logger.info("Post Request Unsuccessful")
             return False
     except OSError as e:
-        print("Warning: " + str(e))
```
Comment (comptonben) [inline uPy/post.py]: print this out too

File: uPy/wifi_connect.py [hunk 3]
```diff
@@ -13,14 +13,19 @@
 from network import WLAN
 from usocket import getaddrinfo
 
+import logging
 import reqst
 
-def station_connected(station: WLAN):
-    print("Connected...Testing Access...")
```
Comment (comptonben) [inline uPy/wifi_connect.py]: print this too

**Pull request merged**